modules/olm-pruning-index-image.adoc

```diff
@@ -41,6 +41,12 @@ endif::[]
 * `podman` version 1.9.3+
 * link:https://github.com/fullstorydev/grpcurl[`grpcurl`]
 * `opm` version 1.12.3+
++
```

Comment:
II had exactly the same thought. Hence I checked with QE and SME if there's a scenario that the user may not refer to opm installation ever and directly get to this step to which I got a reply that it could happen. So thought this would help the users.